$OpenBSD: patch-src_pyoggstreamstate_c,v 1.1 2011/05/30 15:10:17 dcoppa Exp $

Fix memory allocation/deallocation for Python 2.5 and above
(from Debian's patchset for dev-python/pyogg)

--- src/pyoggstreamstate.c.orig	Mon Oct  7 01:04:57 2002
+++ src/pyoggstreamstate.c	Tue Jun  1 15:38:26 2010
@@ -77,7 +77,7 @@ static void 
 py_ogg_stream_state_dealloc(PyObject *self)
 {
   ogg_stream_clear(PY_OGG_STREAM(self));
-  PyMem_DEL(self);
+  PyObject_Del(self);
 }
 
 static PyObject* 
